platforms/hyperledger-fabric/configuration/samples/network-fabricv-add-peer.yaml

```diff
@@ -0,0 +1,558 @@
+      # Do not check-in git_access_token
+      gitops:
+        git_ssh: "ssh://git@github.com/<username>/blockchain-automation-framework.git"         # Gitops ssh url for flux value files 
+        branch: "develop"           # Git branch where release is being made
+        release_dir: "platforms/hyperledger-fabric/releases/dev" # Relative Path in the Git repo for flux sync per environment. 
+        chart_source: "platforms/hyperledger-fabric/charts"     # Relative Path where the Helm charts are stored in Git repo
+        git_push_url: "github.com/<username>/blockchain-automation-framework.git"   # Gitops https URL for git push  (without https://)
+        username: "git_username"          # Git Service user who has rights to check-in in all branches
+        password: "git_access_token"          # Git Server user password
+        email: "git_email"                # Email to use in git config
+        private_key: "path_to_private_key"          # Path to private key file which has write-access to the git repo
+
+      services:
+        ca:
+          name: ca
+          subject: "/C=US/ST=Massachusetts/L=Boston/O=Warehouse/CN=ca.warehouse-net"
+          type: ca
+          grpc:
+            port: 7054
+        peers:
+        - peer:
+          name: peer0          
+          type: anchor    # This can be anchor/nonanchor. Atleast one peer should be anchor peer.   
+          gossippeeraddress: peer0.warehouse-net:7051 # Internal Address of the other peer in same Org for gossip, same peer if there is only one peer
+          peerAddress: peer0.warehouse-net.org3ambassador.blockchaincloudpoc.com:8443 # External URI of the peer 
+          peerstatus: existing      
+          cli: disabled      # Creates a peer cli pod depending upon the (enabled/disabled) tag.   
+          grpc:
+            port: 7051       
+          events:
+            port: 7053
+          couchdb:
+            port: 5984
+          restserver:
+            targetPort: 20001
+            port: 20001 
+          expressapi:
+            targetPort: 3000
+            port: 3000
+          chaincode:
+            name: "chaincode_name"     #This has to be replaced with the name of the chaincode
+            version: "chaincode_version"     #This has to be replaced with the version of the chaincode
+            maindirectory: "chaincode_main"   #The main directory where chaincode is needed to be placed
+            repository:
+              username: "git_username"          # Git Service user who has rights to check-in in all branches
+              password: "git_access_token"
+              url: "github.com/<username>/blockchain-automation-framework.git"
+              branch: develop
+              path: "chaincode_src"   #The path to the chaincode 
+            arguments: 'chaincode_args'     #Arguments to be passed along with the chaincode parameters
+            endorsements: ""        #Endorsements (if any) provided along with the chaincode
```

Comment:
Only the organization having the extra peer should be in this network.yaml

Reply:
yes, addition of a new peer works with a single organization in network.yaml with existing and new peers in it. Tested it. Will change the respective network.yaml